examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,527 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eca904da",
+   "metadata": {},
+   "source": [
+    "## Packages and imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93cdcd4d",
+   "metadata": {},
+   "source": [
+    "We'll begin by installing some analytics packages that we'll use for our examples. We'll also need two packages that allow us to interact with and access Microsoft Azure's storage platform. We'll need:\n",
+    "\n",
+    "* `arrow`\n",
+    "* `ggplot2`\n",
+    "* `jsonlite`\n",
+    "* `geojsonio`\n",
+    "* `sf`\n",
+    "* `tidyverse`\n",
+    "* `AzureStor`\n",
+    "* `AzureRMR`\n",
+    "\n",
+    "which you can install by running `install.packages(c(\"AzureStor\", \"AzureRMR\", \"arrow\", \"ggplot2\", \"jsonlite\", \"geojsonio\", \"sf\", \"tidyverse\")`. These packages allow us to interact with Microsoft Azure, load parquet data, interact with and format data the way we want, and visualize our result. If you're using a different cloud platform for file storage and management, you may need other packages instead of these, but for our demo, we'll use these two to handle connecting with Azure.\n",
+    "\n",
+    "While common practice is to use the `rgdal` package for spatial analytics purposes (see last example), `rgdal` will reach its end-of-supported lifecycle in early 2023. Hence, we recommend the new industry standard `sf` package (standing for \"simple features\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f2e39b1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevant packages\n",
+    "library(arrow)\n",
+    "library(jsonlite)\n",
+    "library(AzureStor)\n",
+    "library(AzureRMR)\n",
+    "library(sf)\n",
+    "library(geojsonio)\n",
+    "library(tidyverse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "acdcd7b9",
+   "metadata": {},
+   "source": [
+    "## Configuring variables and our notebook environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9a58077",
+   "metadata": {},
+   "source": [
+    "Before we get into the code proper, we need to configure a few variables so that things run smoothly. There are four variables that correspond to accessing a cloud-hosted platform directly through R. They will, respectively, allow us to make an authorized connection, to a particular storage provider, so that we can access the endpoint where the files live, within a specific storage container. For this particular implementation, we're hosting files in an Azure Gen. 2 storage bucket, but the principles are the same regardless of the hosting platform."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "43223098",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TENANT_ID = \"YOUR TENANT ID STRING\"\n",
+    "HOST = \"URL TO YOUR STORAGE PROVIDER\" # for example, https://storage.azure.com\n",
```

Comment:
I think this URL needs to end with a forward slash. Can you please include that in your commented example, and maybe note it in parentheses.